mlprimitives/jsons/sklearn.linear_model.LogisticRegression.json

```diff
@@ -0,0 +1,181 @@
+{
+    "name": "sklearn.linear_model.LogisticRegression",
+    "contributors": [
+        "Van Coykendall <vcoyk@mit.edu>",
+        "Hector Dominguez <hedb_12@hotmail.com>"
+    ],
+    "documentation": "https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html",
+    "description": "Logistic Regression (aka logit, MaxEnt) classifier.",
+    "classifiers": {
+        "type": "estimator",
+        "subtype": "classifier"
+    },
+    "modalities": [],
+    "primitive": "sklearn.linear_model.LogisticRegression",
+    "fit": {
+        "method": "fit",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            },
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "produce": {
+        "method": "predict",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "hyperparameters": {
+        "fixed": {
+            "n_jobs": {
+                "type": "int",
+                "description": "Number of CPU cores used when parallelizing over classes if multi_class='ovr'. This parameter is ignored when the 'solver' is set to 'liblinear' regardless of whether 'multi_class' is specified or not. None means 1 unless in a joblib.parallel_backend context. -1 means using all processors",
+                "default": null
+            },
+            "warm_start": {
+                "type": "bool",
+                "description": "When set to True, reuse the solution of the previous call to fit as initialization, otherwise, just erase the previous solution. Useless for liblinear solver.",
+                "default": false
+            },
+            "random_state": {
+                "type": "multitype",
+                "description": "The seed of the pseudo random number generator to use when shuffling the data.",
+                "types": {
+                    "int": {
+                        "description": "random_state is the seed used by the random number generator"
+                    },
+                    "RandomState": {
+                        "description": "random_state is the random number generator."
+                    },
+                    "NoneType": {
+                        "description": "If none, the random number generator is the RandomState instance used by np.random"
+                    }
+                },
+                "default": null
+            },
+            "verbose": {
+                "type": "int",
+                "description": "For the liblinear and lbfgs solvers set verbose to any positive number for verbosity.",
+                "default": 0,
+                "range": [
```

Comment:
Since this can only have two values (0, or "any positive number") and it accepts being passed True or False instead of an integer, let's change this to a bool argument:

```
            "verbose": {
                "type": "bool",
                "description": "For the liblinear and lbfgs solvers set verbose to true for verbosity.",
                "default": false
            },
```

Reply:
4db15d1ee37462acf6b54de5f0df6cd094fe4749 & fdeb881